apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-checkbox-error-handling.component.html

```diff
@@ -0,0 +1,45 @@
+<h3>Checkbox Error Handling</h3>
+<fdp-form-group #fg [hintPlacement]="'left'" [multiLayout]="true" [formGroup]="customForm" [object]="data">
+    <fdp-form-field
+        #ffr1
+        [id]="'presence'"
+        [validators]="validators"
+        [label]="'Your presence is necessary'"
+        [hint]="'presence is necessary'"
+        [hintPlacement]="'left'"
+        [rank]="10"
+        zone="zLeft"
+    >
+        <fdp-checkbox [name]="'presence-cb'" [isBinary]="true" [formControl]="ffr1.formControl"></fdp-checkbox>
+    </fdp-form-field>
+
+    <fdp-form-field
+        #ffr2
+        [id]="'aggrement'"
+        [validators]="validators"
+        [hint]="'this selection is mandatory'"
+        [hintPlacement]="'left'"
+        [rank]="15"
+        zone="zRight"
+    >
+        <fdp-checkbox [name]="'aggrement-cb'" [isBinary]="true" [formControl]="ffr2.formControl">
+            Do you Agree with <a href="#"> Terms and Conditions</a>?
+        </fdp-checkbox>
+    </fdp-form-field>
+
+    <fdp-form-field [id]="'button-1'" [rank]="20" zone="zRight">
+        <fdp-button [type]="'emphasized'" (click)="onSubmit()">Submit</fdp-button>
+    </fdp-form-field>
+
+    <ng-template #i18n let-errors>
+        <span *ngIf="errors.required">
+            Value is required
+        </span>
+        <span *ngIf="errors.invalid">
+            Value is invalid
+        </span>
+    </ng-template>
```

Comment:
Example with errors inside message works weird, could you change trigger events to [focus ,blur] inside of click?

Reply:
have to work separately on form group and form field to resolve trigger issue.